File: atcoder.jp/contests/abc183/tasks/abc183_a/main.cpp
#include <bits/stdc++.h>
using namespace std;

int relu(int x) {
  if (x >= 0) return x;
  return 0;
}

int main() {
  int x;
  cin >> x;

  int ans = relu(x);

  cout << ans << endl;
}

File: atcoder.jp/contests/abc183/tasks/abc183_a/main.go
package main

import (
	"fmt"
)

func relu(x int) int {
	if x >= 0 {
		return x
	}
	return 0
}

func main() {
	var x int
	fmt.Scanf("%d", &x)

	ans := relu(x)

	fmt.Println(ans)
}
